application.py

```diff
@@ -26,9 +26,7 @@ def predictions(district_number):
     prediction = Prediction(db).find(dict(
         district_number=district_number,
         dollars_spent=request.form.get('dollars_spent', None),
-        election_type=request.form.get('election_type'),
```

Comment:
I don't see `election_type` and `is_encumbent` used, based on the predictions csv, so I got rid of these.